README.md

```diff
@@ -47,7 +47,7 @@ $ cargo add duplexify
 ```
 
 ## Safety
-This crate uses ``#![deny(unsafe_code)]`` to ensure everything is implemented in
```

Comment:
Is this particular change needed? Rustc docs and the nomicon both refer specifically to this `#![forbid(unsafe_code)]` invocation